trae_api/core/resources.py

```
@@ -0,0 +1,303 @@
+                
+                # Calculate duration
+                duration_ms = (time.time() - start_time) * 1000
+                
+                # Record response size if we have metrics
+                if metrics and hasattr(result, 'model_dump_json'):
+                    try:
+                        response_size = len(
+                            result.model_dump_json().encode('utf-8')
+                        )
+                        metrics.record_response_size(response_size)
+                    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)